lib/sequent/core/event_store.rb

```diff
@@ -51,7 +51,36 @@ def commit_events(command, streams_with_events)
       end
 
       ##
-      # Returns all events for the aggregate ordered by sequence_number
+      # Returns all events for the aggregate ordered by sequence_number, disregarding snapshot events
+      #
+      def stream_events_for_aggregate(aggregate_id, load_until: nil)
```

Comment:
I think it is good to add the `&block` parameter as well since that is required.